src/ext/oblivc/obliv.h

```diff
@@ -32,5 +33,6 @@ bool execNpProtocol_Bcast1(ProtocolDesc* pd, protocol_run start, void* arg);
 void execNnobProtocol(ProtocolDesc* pd, protocol_run start, void* arg, int numOTs, bool useAltOTExt);
 
 size_t tcp2PBytesSent(ProtocolDesc* pd);
+int tcp2PFlushCount(ProtocolDesc* pd);
```

Comment:
New commit. Even better if commit message has a short 2-3 lines about why this was useful.